### PR TITLE
Adds several new phobias

### PR DESCRIPTION
**Pull request by Anonmare**
[Changelogs]: Adds ~~four~~ ~~five~~ several new phobias to the ones you might receive, careful, you don't want to be a cultist afraid of your own God now, do you?

:cl: 
add: Nanotrasen psychologists have identified new phobias emerging amongst the workforce. Nanotrasen's surgeon general advises all personnel to just buck up and deal with it.
/:cl:

[why]: Because I think it would be fun and makes traumas more... Traumatising?

### CODE REVIEW

File: code/controllers/subsystem/traumas.dm [hunk 1]
```diff
@@ -20,13 +20,19 @@ SUBSYSTEM_DEF(traumas)
...
 
 	phobia_mobs = list("spiders"  = typecacheof(list(/mob/living/simple_animal/hostile/poison/giant_spider)),
 					   "security" = typecacheof(list(/mob/living/simple_animal/bot/secbot)),
 					   "lizards"  = typecacheof(list(/mob/living/simple_animal/hostile/lizard)),
 					   "snakes"   = typecacheof(list(/mob/living/simple_animal/hostile/retaliate/poison/snake))
+					   "robots"   = typecacheof(list(/mob/living/silicon/robot, /mob/living/silicon/AI))
+					   "the supernatural"   = typecacheof(list(/mob/living/simple_animal/hostile/construct, /mob/living/simple_animal/hostile/clockwork/))
 					   )
 
 	phobia_objs = list("spiders"   = typecacheof(list(/obj/structure/spider)),
```
Comment (optimumtact) [inline code/controllers/subsystem/traumas.dm]: this is a very garbage thing

File: code/controllers/subsystem/traumas.dm [hunk 2]
```diff
@@ -20,13 +20,19 @@ SUBSYSTEM_DEF(traumas)
...
+						"doctors"	= strings(PHOBIA_FILE, "doctors"),
+						"authority"	= strings(PHOBIA_FILE, "authority"),
+						"the supernatural"	= strings(PHOBIA_FILE, "the supernatural")
 					   )
 
 	phobia_mobs = list("spiders"  = typecacheof(list(/mob/living/simple_animal/hostile/poison/giant_spider)),
 					   "security" = typecacheof(list(/mob/living/simple_animal/bot/secbot)),
 					   "lizards"  = typecacheof(list(/mob/living/simple_animal/hostile/lizard)),
 					   "snakes"   = typecacheof(list(/mob/living/simple_animal/hostile/retaliate/poison/snake))
+					   "robots"   = typecacheof(list(/mob/living/silicon/robot, /mob/living/silicon/AI))
```
Comment (ChangelingRain) [inline code/controllers/subsystem/traumas.dm]: this should include drones and things like medibots

File: code/controllers/subsystem/traumas.dm [hunk 3]
```diff
@@ -20,13 +20,19 @@ SUBSYSTEM_DEF(traumas)
...
+					   "the supernatural"   = typecacheof(list(/mob/living/simple_animal/hostile/construct, /mob/living/simple_animal/hostile/clockwork/))
```
Comment (ChangelingRain) [inline code/controllers/subsystem/traumas.dm]: this should include cogscarabs

File: strings/phobia.json [hunk 4]
```diff
@@ -71,5 +71,87 @@
...
+		"nanotrasen",
+		"admiral",
+		"centcom",
+		"comdom"
+	],
+
+	"the supernatural": [
+		"chaplain",
+		"chappy",
+		"chap",
```
Comment (ChangelingRain) [inline strings/phobia.json]: hey, old chap.

File: code/controllers/subsystem/traumas.dm (hunk 1 above)
Comment (Anonmare) [reply code/controllers/subsystem/traumas.dm]: That's a bit out of my scope and experience, I'm working with what's already there unfortunately.

File: strings/phobia.json (hunk 4 above)
Comment (Anonmare) [reply strings/phobia.json]: Fiiine I'll remove that. Damn context-sensitivity...

File: code/controllers/subsystem/traumas.dm [hunk 5]
```diff
@@ -20,35 +20,80 @@ SUBSYSTEM_DEF(traumas)
...
 	phobia_mobs = list("spiders"  = typecacheof(list(/mob/living/simple_animal/hostile/poison/giant_spider)),
 					   "security" = typecacheof(list(/mob/living/simple_animal/bot/secbot)),
 					   "lizards"  = typecacheof(list(/mob/living/simple_animal/hostile/lizard)),
-					   "snakes"   = typecacheof(list(/mob/living/simple_animal/hostile/retaliate/poison/snake))
+					   "snakes"   = typecacheof(list(/mob/living/simple_animal/hostile/retaliate/poison/snake)),
+					   "robots"   = typecacheof(list(/mob/living/silicon/robot, /mob/living/silicon/ai, 
+					   /mob/living/simple_animal/drone, /mob/living/simple_animal/bot/)),
+					   "the supernatural"   = typecacheof(list(/mob/living/simple_animal/hostile/construct, 
+					   /mob/living/simple_animal/hostile/clockwork/, /mob/living/simple_animal/drone/cogscarab))
 					   )
```
Comment (Armhulen) [inline code/controllers/subsystem/traumas.dm]: can you add skeleton simplemobs to this list i keep forgetting to fix this sorry

**Review by ghost — COMMENTED** [review]
please and thank you, while you're here!

File: code/controllers/subsystem/traumas.dm (hunk 5 above)
Comment (Anonmare) [reply code/controllers/subsystem/traumas.dm]: Sure no problem

**Comment by ExcessiveUseOfCobblestone** [comment]
scared?

**Comment by Armhulen** [comment]
infinite thanks anonmareeeee

**Comment by Shadowflame909** [comment]
Remember how bad hissing was. Why would we want the chaplain to be afraid of his job again?

**Comment by Anonmare** [comment]
Technically sec and assistants can get phobias of their own job. Curator started with the snake phobia which was the issue

File: strings/phobia.json [hunk 6]
```diff
@@ -71,5 +71,86 @@
...
+		"hop",
+		"hos",
+		"ce",
+		"cmo",
+		"rd",
+		"command",
+		"nanotrasen",
+		"admiral",
+		"centcom",
+		"comdom"
```
Comment (Iamgoofball) [inline strings/phobia.json]: dude

it's condom

Comment (Bawhoppen) [reply strings/phobia.json]: The whole captain is comdom meme isn't supposed to be correctly spelled

Comment (Iamgoofball) [reply strings/phobia.json]: i'm fairly certain it's always been captain is condom

File: code/controllers/subsystem/traumas.dm [hunk 7]
```diff
@@ -20,35 +20,81 @@ SUBSYSTEM_DEF(traumas)
...
+						/obj/machinery/door/airlock/cult/, /obj/singularity/narsie, /obj/item/device/soulstone,
+						/obj/structure/destructible/clockwork/, /obj/item/clockwork/, /obj/item/clothing/suit/armor/clockwork, 
+						/obj/item/clothing/glasses/judicial_visor, /obj/effect/clockwork/sigil, /obj/item/stack/tile/brass, 
+						/turf/open/floor/clockwork, /turf/closed/wall/clockwork, /obj/machinery/door/airlock/clockwork/,
+						/obj/item/clothing/suit/wizrobe, /obj/item/clothing/head/wizard, /obj/item/spellbook, /obj/item/staff, 
+						/obj/item/clothing/suit/space/hardsuit/shielded/wizard, /obj/item/clothing/suit/space/hardsuit/wizard,
+						/obj/item/gun/magic/staff/, /obj/item/gun/magic/wand/,
+						/obj/item/nullrod, /obj/item/clothing/under/rank/chaplain))
 					   )
 	phobia_turfs = list("space" = typecacheof(list(/turf/open/space, /turf/open/floor/holofloor/space, /turf/open/floor/fakespace)))
```
Comment (XDTM) [inline code/controllers/subsystem/traumas.dm]: Cult/clickwork walls and floor?

File: strings/phobia.json (hunk 6 above)
Comment (PKPenguin321) [reply strings/phobia.json]: It's definitely both

File: code/controllers/subsystem/traumas.dm (hunk 7 above)
Comment (Anonmare) [reply code/controllers/subsystem/traumas.dm]: Clockwork floors and walls are already a phobia. I separated out bloodcult, clockcult, magic and religion for the sake of readability

Comment (XDTM) [reply code/controllers/subsystem/traumas.dm]: Now that i see it you put them in the object list, which won't work and might runtime; move turfs to this section please

Comment (Anonmare) [reply code/controllers/subsystem/traumas.dm]: Got it

**Comment by Anonmare** [comment]
I got bullied by Discord so now there's 5 phobias

**Comment by evsey9** [comment]
give all assistants phobia of sec

File: code/controllers/subsystem/traumas.dm [hunk 8]
```diff
@@ -20,35 +22,106 @@ SUBSYSTEM_DEF(traumas)
...
+						/obj/item/retractor, /obj/item/hemostat, /obj/item/cautery, /obj/item/surgicaldrill, /obj/item/scalpel, /obj/item/circular_saw)),
+						
+						"authority"   = typecacheof(list(/obj/item/clothing/under/rank/captain,  /obj/item/clothing/under/rank/head_of_personnel, 
+						/obj/item/clothing/under/rank/head_of_security, /obj/item/clothing/under/rank/research_director, 
+						/obj/item/clothing/under/rank/chief_medical_officer, /obj/item/clothing/under/rank/chief_engineer, 
+						/obj/item/clothing/under/rank/centcom_officer, /obj/item/clothing/under/rank/centcom_commander, 
+						/obj/item/melee/classic_baton/telescopic, /obj/item/card/id/silver, /obj/item/card/id/gold, 
+						/obj/item/card/id/captains_spare, /obj/item/card/id/centcom, /obj/machinery/door/airlock/command)),
+						
+						"the supernatural"  = typecacheof(list(/obj/structure/destructible/cult, /obj/item/tome,
```
Comment (ShizCalev) [inline code/controllers/subsystem/traumas.dm]: add toys too imo, `/obj/item/toy/clockwork_watch`, `/obj/item/toy/toy_dagger`, `/obj/item/toy/plush/plushvar`, `/obj/item/toy/plush/narplush`

File: code/controllers/subsystem/traumas.dm [hunk 9]
```diff
@@ -20,35 +22,106 @@ SUBSYSTEM_DEF(traumas)
...
+						/obj/item/device/soulstone/,
+						/obj/structure/destructible/clockwork/, /obj/item/clockwork/, /obj/item/clothing/suit/armor/clockwork, 
+						/obj/item/clothing/glasses/judicial_visor, /obj/effect/clockwork/sigil/, /obj/item/stack/tile/brass, 
+						/obj/machinery/door/airlock/clockwork/,
+						/obj/item/clothing/suit/wizrobe, /obj/item/clothing/head/wizard, /obj/item/spellbook, /obj/item/staff, 
+						/obj/item/clothing/suit/space/hardsuit/shielded/wizard, /obj/item/clothing/suit/space/hardsuit/wizard,
+						/obj/item/gun/magic/staff/, /obj/item/gun/magic/wand/,
+						/obj/item/nullrod, /obj/item/clothing/under/rank/chaplain)),
+						
+						"aliens"   = typecacheof(list(/obj/item/clothing/mask/facehugger, /obj/item/organ/body_egg/alien_embryo,
```
Comment (ShizCalev) [inline code/controllers/subsystem/traumas.dm]: `/obj/item/toy/toy_xeno`

File: code/controllers/subsystem/traumas.dm [hunk 10]
```diff
@@ -20,35 +22,106 @@ SUBSYSTEM_DEF(traumas)
...
+						/turf/open/floor/plasteel/cult, /turf/closed/wall/mineral/cult)),
+						"aliens" = typecacheof(list(/turf/open/floor/plating/abductor, /turf/open/floor/plating/abductor2, 
+						/turf/open/floor/mineral/abductor, /turf/closed/wall/mineral/abductor))
+						)
+
+	phobia_species = list("lizards"   = typecacheof(list(/datum/species/lizard/)),
+						  "skeletons" = typecacheof(list(/datum/species/skeleton, /datum/species/plasmaman)),
+						  "robots"   = typecacheof(list(/datum/species/android)),
+						  "the supernatural" = typecacheof(list(/datum/species/golem/clockwork, /datum/species/golem/runic)),
+						  "aliens" = typecacheof(list(/datum/species/abductor, /datum/species/jelly/, /datum/species/pod,
```
Comment (coiax) [inline code/controllers/subsystem/traumas.dm]: Trailing / on the jelly.

Comment (Anonmare) [reply code/controllers/subsystem/traumas.dm]: Intended, it's so I don't have to write down each subspecies or make any additions if more jelly subspecies get added

Comment (XDTM) [reply code/controllers/subsystem/traumas.dm]: You don't need the trailing / for that, it works that way by default

Comment (Anonmare) [reply code/controllers/subsystem/traumas.dm]: Oh it does? Very well then

**Pull request merged**